main.nf

```diff
@@ -244,16 +244,16 @@ process picard {
     module 'picard/2.0.1'
     module 'BEDTools'
     module 'samtools'
-    
+
     cpus 2
     memory { 16.GB * task.attempt }
     time { 24.h * task.attempt }
     errorStrategy { task.exitStatus == 143 ? 'retry' : 'terminate' }
     maxRetries 3
     maxErrors '-1'
-    
+
     publishDir "${params.outdir}/picard", mode: 'copy'
-    
+
     input:
     file bam_picard
     set val(prefix) from name_for_picard
```

Comment:
Aha, ok. What's the error message?

On Thu, Jun 16, 2016 at 4:28 PM, Phil Ewels <notifications@github.com>
wrote:

> In main.nf
> <https://github.com/SciLifeLab/NGI-ChIPseq/pull/4#discussion_r67354541>:
>
> >      publishDir "${params.outdir}/picard", mode: 'copy'
> > -
> > +
> >      input:
> >      file bam_picard
> >      set val(prefix) from name_for_picard
>
> Yup, that worked! Looks good now, but there was another bug that was
> different which needed to be fixed after splitting the process into two.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/SciLifeLab/NGI-ChIPseq/pull/4/files/e836f3f74c85ac3dd83cf378873ed5deab42205d#r67354541>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ARdKNI7fkNSVFZ5nEoosMtocKpCBEA50ks5qMV2LgaJpZM4I3XJD>
> .
>